3ds2/src/test/java/com/adyen/checkout/adyen3ds2/Adyen3DS2ComponentTest.kt

```diff
@@ -47,7 +48,7 @@ internal class Adyen3DS2ComponentTest(
 
     @BeforeEach
     fun before() {
-        Logger.setLogcatLevel(Logger.NONE)
+        AdyenLogger.setLogLevel(Logger.NONE)
```

Comment:
Just an idea that doesn't have to do with this PR: we should create our own test extension which does this and possibly other generic setup things